tests/kokkos-based/helpers.hpp

```diff
@@ -84,5 +84,235 @@ auto create_stdvector_and_copy_rowwise(T sourceView)
   return result;
 }
 
+// create rank-1 mdspan (vector)
+template <typename ValueType,
+          typename mdspan_t = typename _blas2_signed_fixture<ValueType>::mdspan_r1_t>
+mdspan_t make_mdspan(ValueType *data, std::size_t ext) {
+  return mdspan_t(data, ext);
+}
+
+template <typename ValueType,
```

Comment:
I don't think you actually need the `const ValueType` overload, as the `ValueType` generic version should deduce `const` as needed.  For example, the following should deduce `const ValueType` as the `element_type` of the `mdspan`.

```c++
template <typename ValueType>
auto make_mdspan(const std::vector<ValueType> &v) {
  return make_mdspan(v.data(), v.size()); // v.data() returns const ValueType*
}
```

Reply:
Correct - thank you for pointing this out! Fixed on https://github.com/kokkos/stdBLAS/pull/187/commits/2f72e982654431235dcd7bd0f6c2f786954ed329.